spec/components/find_interface/results/no_results_component_spec.rb

```diff
@@ -0,0 +1,127 @@
+require "rails_helper"
+
+module FindInterface
+  describe Results::NoResultsComponent, type: :component do
+    it "renders nothing if there are results" do
+      results_view = instance_double(
+        Find::ResultsView,
+        country: "Scotland",
+        devolved_nation?: true,
+        no_results_found?: false,
+      )
+      component = render_inline(described_class.new(results: results_view))
+
+      expect(component.text).to be_blank
+    end
+
+    context "Devolved nations" do
+      it "renders devolved nation warning" do
+        results_view = instance_double(
+          Find::ResultsView,
+          country: "Scotland",
+          devolved_nation?: true,
+          no_results_found?: true,
+        )
+        component = render_inline(described_class.new(results: results_view))
+
+        expect(component.text).to include("This service is for courses in England")
+      end
+
+      context "Scotland" do
+        it "renders Scottish teacher training website" do
+          results_view = instance_double(
+            Find::ResultsView,
+            country: "Scotland",
+            devolved_nation?: true,
+            no_results_found?: true,
+          )
+          component = render_inline(described_class.new(results: results_view))
+
+          expect(component).to have_link("Learn more about teacher training in Scotland", href: "https://teachinscotland.scot/")
+        end
+      end
+
+      context "Wales" do
+        it "renders Scottish teacher training website" do
```

Comment:
Wales was never Scottish

Reply:
```suggestion
        it "renders Welsh teacher training website" do
```

![image](https://user-images.githubusercontent.com/470137/198547417-83f436b2-7704-4b44-8101-fa076f622e61.png)